bin/oneoff/import_concept_difficulty_tags.rb

```diff
@@ -5,37 +5,28 @@
 require 'csv'
 
 # Import level concept difficulty tags from a gsheet (downloaded as a series of
-# CSVs). When we cloned the CSF courses from 2017 to 2018, the LCDs did not get
+# CSVs). When we cloned the CSF courses from 2017 to 2018 and 2018 to 2019, the LCDs did not get
 # cloned with them so we're doing a mass import.
 
-# The given CSV contained headers that didn't map exactly to our defined LCDs,
-# so we perform a simple mapping in code
-HEADERS_TO_CONCEPTS = {
-  sequence_algorithms: ConceptDifficulties::SEQUENCING,
-  repeat: ConceptDifficulties::REPEAT_LOOPS,
-  functions_wparams: ConceptDifficulties::FUNCTIONS_WITH_PARAMS
-}
-
 def main(csv_dir)
-  %w(CourseA CourseB CourseC CourseD CourseE CourseF).each do |course|
+  %w(CourseA CourseB CourseC CourseD CourseE CourseF PreExpress Express).each do |course|
     CSV.foreach("#{csv_dir}/#{course}.csv", headers: true,  header_converters: :symbol) do |row|
-      concept_difficulties = Hash[row.to_hash.compact.map do |key, val|
-        [HEADERS_TO_CONCEPTS.fetch(key, key.to_s), val]
-      end].delete_if do |key, _val|
-        !LevelConceptDifficulty::CONCEPTS.include?(key)
-      end
-
       level = Level.find_by(name: row.fetch(:level_name))
       unless level
         puts "cannot find #{row.fetch(:level_name)}"
         next
       end
 
+      concept_difficulties = row.to_hash.compact.delete_if do |key, _val|
+        !LevelConceptDifficulty::CONCEPTS.include?(key.to_s)
```

Comment:
would it be valuable to warn about invalid keys here?

Reply:
yeah, good call. we will always have some invalid keys here (script, stage, position, level_name, and type are the additional keys in 2019's version of this CSV), so we could whitelist those keys and warn about anything outside of `WHITELISTED_KEYS` & `LevelConceptDifficulty::CONCEPTS`